pkg/codegen/docs/gen.go

```diff
@@ -896,7 +896,8 @@ func (mod *modContext) genConstructorPython(r *schema.Resource, argsOptional, ar
 		return getDockerImagePythonFormalParams()
 	}
 
-	var params []formalParam
+	// We perform at least three appends before iterating over input types.
+	params := make([]formalParam, 0, 3+len(mod.inputTypes))
```

Comment:
```suggestion
	params := make([]formalParam, 0, 3+len(r.InputProperties))
```

iteration over mod.inputTypes is not the one doing the appends